kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateListRow.js

```diff
@@ -134,7 +134,7 @@ define([
 
             // add the log widget to the next `tr` element
             widgets.log = JobLogViewer.make({ showHistory: true });
-            widgets.log.start({
+            await widgets.log.start({
```

Comment:
made starting the JobLogViewer widget async (it makes testing easier, amongst other things!)